src/main/java/com/dougnoel/sentinel/steps/CsvSteps.java

```diff
@@ -0,0 +1,142 @@
+package com.dougnoel.sentinel.steps;
+
+import com.dougnoel.sentinel.configurations.Configuration;
+import com.dougnoel.sentinel.exceptions.FileException;
+import com.dougnoel.sentinel.files.CsvFile;
+import com.dougnoel.sentinel.strings.SentinelStringUtils;
+import com.dougnoel.sentinel.system.FileManager;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+
+import static org.junit.Assert.*;
```

Comment:
To my eye we've only got at most 2 from a class in the above imports. Is that enough to warrant using the *? Personally I never really notice import sections because the IDE collapses them, but if we have some standard I'm happy to abide by it.